docs/guides/2-cli.md

```diff
@@ -44,7 +44,8 @@ The Spectral CLI supports loading documents as YAML or JSON, and validation of O
 
 ## Using a Ruleset File
 
-If you don't specify a ruleset file with the `--ruleset` parameter, the Spectral CLI will look for a ruleset file called `.spectral.yml`, `.spectral.yaml`, or `.spectral.json` in the current working directory. If no ruleset is specified and no default ruleset file is found, the built-in rulesets will be used.
+If you don't specify a ruleset file with the `--ruleset` parameter, the Spectral CLI will look for a ruleset file called `.spectral.yml`, `.spectral.yaml`, `.spectral.json` or `.spectral.js` in the current working directory.
+Spectral will refuse to lint the document if no ruleset is specified and no default ruleset file is found.
```

Comment:
If this is meant to be a new paragraph it will need another line.

Reply:
We can have a separate paragraph, sure.
```suggestion

Spectral will refuse to lint the document if no ruleset is specified and no default ruleset file is found.
If you want to have a simple basic ruleset for OpenAPI, this command will let you get started:
`echo "extends: spectral:oas" > .spectral.yaml` 
```

@philsturgeon thoughts?

Reply:
I just ran up against this change and found this confusing behaviour.  A few other people in https://github.com/stoplightio/spectral/issues/1796 had a similar experience from the sound of things.

Reply:
Ooops, I lost sight of these comments.

> Is this new behavior in Spectral v6? We used to autodetect and apply OAS if we thought it was OAS, but now folks have to specify that for theirselves?

Yeah, it's a deliberate change we've made.
Overall, from a technical standpoint it is quite trivial to load some default ruleset - would probably be like a one-liner kind of a change if we were to make it, so totally straightforward to do.
Our goal was to convince users to write their own style guides, but I'm all in for constant reiteration of the standpoint if this happens to be truly annoying - I believe that most of folks do have some kind of a ruleset, so this change is  seamless for them.

Reply:
Ok, thanks for clarifying.